include/ObjectIntrospection.h

```diff
@@ -139,25 +141,25 @@ class CodegenHandler {
     delete lib;
   }
 
-  static int getObjectSize(T *ObjectAddr, size_t *ObjectSize) {
+  static int getObjectSize(T &ObjectAddr, size_t &ObjectSize) {
```

Comment:
Can we make these ObjectAddrs const references? (and the same for the other functions)

```suggestion
  static int getObjectSize(const T &ObjectAddr, size_t &ObjectSize) {
```

Reply:
It seems like we can. New commit added to this diff doing that.